hugr-model/src/v0/mod.rs

```diff
@@ -48,15 +48,39 @@
 //! development. However, there are still some mismatches with `hugr-core` that are not
 //! addressed by conversions in import/export:
 //!
-//! - Some static types can not yet be represented in `hugr-core` although they should be.
+//! - Some terms can not yet be represented in `hugr-core` although they should be.
+//!   Importing such terms will result in an error that declares these terms as currently
+//!   unsupported.
+//! - In particular `hugr-core` currently only allows to define custom runtime types but
```

Comment:
This _will_ get outdated, so it'd be nice to pin a version
```suggestion
//! - In particular `hugr-core` (as of `v0.22.3`) only allows to define custom runtime types but
```